File: web/starter/src/app/dto/propuesta-create-dto.ts
export class PropuestaCreateDto {
    titulo: String;
    contenido: string;
    creador: String;
    partido: String;
    materia: String;

    constructor(    titulo: String,
        contenido: string,
        creador: String, partido: String, materia: String) {
            this.titulo = titulo;
            this.contenido = contenido;
            this.creador = creador;
            this.partido = partido;
            this.materia = materia;
        }
}

File: mobile/app/src/main/java/dam/javazquez/tuvotocuenta/dto/PropuestaCreateDto.java
package dam.javazquez.tuvotocuenta.dto;

import dam.javazquez.tuvotocuenta.responses.PropuestaResponse;

public class PropuestaCreateDto {

    private String titulo;
    private String contenido;
    private String materia;
    private String partido;

    public PropuestaCreateDto() {

    }

    public PropuestaCreateDto(String titulo, String contenido, String materia, String partido) {
        this.titulo = titulo;
        this.contenido = contenido;
        this.materia = materia;
        this.partido = partido;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getContenido() {
        return contenido;
    }

    public void setContenido(String contenido) {
        this.contenido = contenido;
    }

    public String getMateria() {
        return materia;
    }

    public void setMateria(String materia) {
        this.materia = materia;
    }

    public String getPartido() {
        return partido;
    }

    public void setPartido(String partido) {
        this.partido = partido;
    }
}
